wsdl_docs.inc

```diff
@@ -631,7 +631,14 @@ function _wsdl_docs_service_import($name) {
     return 'ERROR input file is not XML';
   }
   // Update existing service with new specs per file upload.
-  wsdl_docs_update_wsclient_service($service, $DOMDocument);
-
+  // wsclient_soap.module function initializeMetadata() can throw an
```

Comment:
I put a note in here because my IDE is complaining about a try/catch block here - I'm guessing this is because the PHPDoc comment for the wsclient_soap function that actually throws the exception doesn't declare `@throws` in the PHPDoc...